app/views/user_mailer/shared/_in_person_ready_to_verify.html.erb

```diff
@@ -84,7 +84,7 @@
       <%= t('date.day_names')[6] %>: <%= @presenter.selected_location_hours(:sunday) %>
     </div>
     <div>
-      <%= @presenter.selected_location_details[:phone] %>
+      <%= @presenter.selected_location_details['phone'] %>
```

Comment:
I've seen we've had a few similar instances of this sort of problem since it was introduced. I wonder if we could make this hash a [HashWithIndifferentAccess](https://api.rubyonrails.org/v4.2/classes/ActiveSupport/HashWithIndifferentAccess.html) so that it would support keys referenced either as strings or as symbols. It's complicated by the fact that this is a database field, but maybe it's possible to override the method on the [`InPersonEnrollment` class](https://github.com/18F/identity-idp/blob/main/app/models/in_person_enrollment.rb)?